pkgs/applications/misc/rofi-screenshot/deps.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/rofi-screenshot b/rofi-screenshot
```

Comment:
Any reason why this patch should be shipped? It looks like something that will often be the source of merge conflicts, and it's useful to check whether the derivation includes everything it needs in the wrapped $PATH.

Reply:
Well, it's unnecessary runtime checks. But sure, reverted.